src/js/letters/containers/Main.jsx

```diff
@@ -56,36 +56,18 @@ export class Main extends React.Component {
       case awaitingResponse:
         appContent = <LoadingIndicator message="Loading your letters..."/>;
         break;
-      case backendServiceError:
-        appContent = systemDownMessage;
-        break;
       case backendAuthenticationError:
         appContent = recordsNotFound;
         break;
-      case invalidAddressProperty:
-        appContent = systemDownMessage;
-        break;
       case letterEligibilityError:
         appContent = this.props.children;
         break;
-      case unavailable:
-        appContent = (
-          <div id="lettersUnavailable">
-            <div className="usa-alert usa-alert-error" role="alert">
-              <div className="usa-alert-body">
-                <h4 className="usa-alert-heading">Letters Unavailable</h4>
-                <p className="usa-alert-text">
-                  We weren’t able to retrieve your VA letters. Please call <a href="tel:855-574-7286">1-855-574-7286</a>,
-                  TTY: <a href="tel:18008778339">1-800-877-8339</a>, Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (ET).
-                </p>
-              </div>
-            </div>
-            <br/>
-          </div>
-        );
-        break;
+      case unavailable: // fall-through to default
```

Comment:
Per chat with Peggy & Alex. Though as a follow-up we may want to tweak `SystemDown` error message globally to get it more in line with current content direction.